apps/web/src/styles/toast.css

```
@@ -83,6 +193,11 @@
 }
 
 .toast.toast-info {
+  --toast-accent: #0284c7;
+  border-color: rgba(3, 105, 161, 0.28);
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)